### ISSUE TITLE
Add get_conduit_data helper method

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#1754]
Many of the models request both pore and throat properties for the same thing, like `pore_temperature` and `throat_temperature`.  It would be very handy to offer a shortcut for this, like `get_conduit_data('temperature')` that would return an Nt-by-3 array of temperatures in each element of each conduit. 

The name and api are open to debate of course.

**Comment by jgostick** [comment]
So a model definition would be like:

``` python
def viscosity(target, temperature='temperature', density='density'):
    T_in_pore1, T_in_throat, T_in_pore_2 = target.get_conduit_data(temperature)
    mu_in_pore1 = f(T_in_pore1)
    <etc>
```

**Comment by jgostick** [comment]
We *could* also update the __getitem__ to return conduit values if 'conduit' is requested, like:

``` python
    T_in_pore1, T_in_throat, T_in_pore_2 = target['conduit.'+temperature]
```

The above would be convenient but maybe bit to much hidden magic?

**Comment by jgostick** [comment]
We could make the 'getter' method more general so that it interpolates if needed, so:

``` python
   T_in_throat = target.get_throat_data('temperature')
```

Would do interpolation to find throat temperatures if not present, otherwise could just return the values.

**Comment by jgostick** [comment]
closed in PR #1996
